zh-cn: update the description of the `animationiteration` event

kongbai1996 opened the pull request:
Updated the Chinese translation of the animationend event description.

<!-- 🙌 Thanks for contributing to MDN Web Docs. Adding details below will help us to merge your PR faster. -->

### Description

Updated the Chinese translation of the animationend event description.

### Motivation

The previous translation is wrong, which is easy to cause readers to misunderstand.

### Additional details

https://developer.mozilla.org/en-US/docs/Web/API/Element/animationiteration_event

### Related issues and pull requests

N/A

kongbai1996 committed:
Update Chinese translation

Updated the Chinese translation of the animationend event description.

kongbai1996 committed:
Updated the Chinese translation of the animationend event description.

Updated the Chinese translation of the animationend event description.

yin1999 commented on files/zh-cn/web/api/element/animationiteration_event/index.md:
```diff
@@ -5,7 +5,7 @@ slug: Web/API/Element/animationiteration_event
 {{APIRef("CSS3 Animations")}}
 
-[`animationiteration`](/zh-CN/docs/Web/API/Element/animationiteration_event)事件的处理器 . 当 [CSS Animation](/zh-CN/docs/Web/CSS/CSS_animations) 运动到最后一帧时触发。An iteration ends when a single pass through the sequence of animation instructions is completed by executing the last animation step.
+ **`animationiteration`** 事件的处理器。当一个[CSS Animation](/zh-CN/docs/Web/CSS/CSS_animations)的迭代结束，另一个开始时，会触发[`animationiteration`](/zh-CN/docs/Web/API/Element/animationiteration_event)事件。此事件不会与[`animationend_event`](/zh-CN/docs/Web/API/Element/animationend_event)事件同时发生，因此对于动画迭代计数为1的动画不会发生。
```
请按照[英文文档](https://developer.mozilla.org/en-US/docs/Web/API/Element/animationiteration_event)去修改翻译。这段翻译已经过时了。（现在文档的 URL 是 `Web/API/Element/animationiteration_event`，即当前文档应该描述这个事件，而非事件处理器属性）

kongbai1996 replied:
已经把多余的语句删除了

kongbai1996 committed:
Update index.md

Updated the Chinese translation of the animationend event description.

kongbai1996 committed:
Updated the Chinese translation of the animationend event description.Update files/zh-cn/web/api/element/animationiteration_event/index.md

Updated the Chinese translation of the animationend event description. by reviewdog.

Co-authored-by: github-actions[bot] <41898282+github-actions[bot]@users.noreply.github.com>

kongbai1996 committed:
Merge branch 'main' into kongbai1996-chinese-translation

yin1999 committed:
Update index.md

yin1999 reviewed (approved):
感谢 @kongbai1996。修改后的内容还是过时的，我对照英文文档更新了一下。

kongbai1996 commented:
> 感谢 @kongbai1996。修改后的内容还是过时的，我对照英文文档更新了一下。
你是怎么在我的PR上面追加你的修改的呀，小白求教，甩个文档学习学习
